lib/classifier-reborn/backends/bayes_redis_backend.rb

```diff
@@ -1,4 +1,8 @@
-require 'redis'
+begin
+  require 'redis'
+rescue LoadError
+  puts 'The redis gem is required to use the redis backend.'
```

Comment:
Ok, I'll update this to raise with a message including the gem name and maybe some relevant info. I'll play around with error messages and see if I can come up with something intuitive and helpful.